Source/UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -773,6 +774,37 @@ - (void)testCallbackQueue_WithoutFetcherService {
   [self testCallbackQueue];
 }
 
+- (void)testCallbackQueue_SerialDispatch {
+  if (!_isServerRunning) return;
+
```

Comment:
```
CREATE_START_STOP_NOTIFICATION_EXPECTATIONS(1, 1);
```

Sorry, this is an unfortunate thing that's required due to many tests having `NSNotification` asserts and the non-main queue for the delegate methods.

Reply:
Done.